<ui version="4.0" >
 <class>BookmarksManagerDlg</class>
 <widget class="QDialog" name="BookmarksManagerDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Bookmarks Manager Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Bookmarks List</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="bookmarksList" >
        <property name="showDropIndicator" stdset="0" >
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors" >
         <bool>false</bool>
        </property>
        <property name="sortingEnabled" >
         <bool>false</bool>
        </property>
        <property name="animated" >
         <bool>false</bool>
        </property>
        <property name="allColumnsShowFocus" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>Double click an item to open the url in your browser or join the group chat via Psi.</string>
        </property>
        <column>
         <property name="text" >
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Url or Jid</string>
         </property>
        </column>
        <item>
         <property name="text" >
          <string>Web Pages:</string>
         </property>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Conferences:</string>
         </property>
         <property name="text" >
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="IconButton" native="1" name="add" >
          <property name="text" >
           <string>Add</string>
          </property>
          <property name="icon" >
           <string>psi/add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="IconButton" native="1" name="modify" >
          <property name="text" >
           <string>Modify</string>
          </property>
          <property name="icon" >
           <string>psi/modify</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="IconButton" native="1" name="remove" >
          <property name="text" >
           <string>Remove</string>
          </property>
          <property name="icon" >
           <string>psi/remove</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title" >
        <string>current Bookmark Details</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item rowspan="3" row="2" column="0" colspan="2" >
         <widget class="QGroupBox" name="groupBox_2" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title" >
           <string>Type</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="0" column="0" >
            <widget class="QRadioButton" name="urlCheck" >
             <property name="text" >
              <string>Url</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QRadioButton" name="conferenceCheck" >
             <property name="text" >
              <string>Conference</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Url or Jid:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" >
         <widget class="QLineEdit" name="username" />
        </item>
        <item row="3" column="3" >
         <widget class="QLineEdit" name="password" />
        </item>
        <item row="4" column="2" colspan="2" >
         <widget class="QCheckBox" name="autoJoin" >
          <property name="text" >
           <string>Autojoin on Psi startup.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="3" >
         <widget class="QLineEdit" name="urlOrJid" />
        </item>
        <item row="0" column="1" colspan="3" >
         <widget class="QLineEdit" name="bookmarkName" />
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Name:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconButton</class>
   <extends>QWidget</extends>
   <header>iconbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>bookmarksList</tabstop>
  <tabstop>add</tabstop>
  <tabstop>modify</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>name</tabstop>
  <tabstop>urlOrJid</tabstop>
  <tabstop>urlCheck</tabstop>
  <tabstop>conferenceCheck</tabstop>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>autoJoin</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>add</sender>
   <signal>clicked()</signal>
   <receiver>BookmarksManagerDlg</receiver>
   <slot>onAdd()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remove</sender>
   <signal>clicked()</signal>
   <receiver>BookmarksManagerDlg</receiver>
   <slot>onRemove()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modify</sender>
   <signal>clicked()</signal>
   <receiver>BookmarksManagerDlg</receiver>
   <slot>onModify()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BookmarksManagerDlg</receiver>
   <slot>onAccept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BookmarksManagerDlg</receiver>
   <slot>onReject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bookmarksList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>BookmarksManagerDlg</receiver>
   <slot>onItemSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bookmarksList</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem *, int)</signal>
   <receiver>BookmarksManagerDlg</receiver>
   <slot>onItemDoubleClicked(QTreeWidgetItem *, int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>urlCheck</sender>
   <signal>clicked()</signal>
   <receiver>BookmarksManagerDlg</receiver>
   <slot>onRBClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>87</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel" >
     <x>237</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>conferenceCheck</sender>
   <signal>clicked()</signal>
   <receiver>BookmarksManagerDlg</receiver>
   <slot>onRBClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>87</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel" >
     <x>237</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>